google/cloud/spanner_v1/pool.py

```diff
@@ -472,11 +503,22 @@ class TransactionPingingPool(PingingPool):
                     by the pool.
     """
 
-    def __init__(self, size=10, default_timeout=10, ping_interval=3000, labels=None):
+    def __init__(
+        self,
+        size=10,
+        default_timeout=10,
+        ping_interval=3000,
+        labels=None,
+        database_role=None,
```

Comment:
The `database_role` argument docstrings should be added everywhere the argument was added